src/handlers/track_table.rs

```diff
@@ -59,12 +59,10 @@ pub fn handler(key: Key, app: &mut App) {
                             }
                         };
                     }
-                    TrackTableContext::RecommendedTracks => {}
                     TrackTableContext::SavedTracks => {
                         app.get_current_user_saved_tracks_next();
                     }
-                    TrackTableContext::AlbumSearch => {}
-                    TrackTableContext::PlaylistSearch => {}
+                    _ => {}
```

Comment:
I tend to prefer to explicitly match on enums to let the compiler complain when a new item is added to the enum that is not yet matched. Could you revert this change and also line 93?